libmamba/src/api/create.cpp

```diff
@@ -64,8 +65,22 @@ namespace mamba
                 }
                 else
                 {
-                    LOG_ERROR << "Non-conda folder exists at prefix";
-                    throw std::runtime_error("Aborting.");
+                    std::stringstream msg;
+
+                    msg << "WARNING: A directory already exists at the target location '";
+                    msg << ctx.prefix_params.target_prefix.string();
+                    msg << "'\nbut it is not a conda environment.\n";
+                    msg << "Continue creating environment?";
+
+                    if (Console::prompt(msg.str(), 'n'))
+                    {
+                        fs::remove_all(ctx.prefix_params.target_prefix);
```

Comment:
An improvement would be to do it transactionally: at the beginning of the function, use `on_scope_exit` with this code with a condition on a boolean which would tell if we need to do this delete. By default we do, except if going in the other case. That way future exceptions/errors will also remove the directory.

But that's a minor improvement here.